### PR TITLE
add state expiry without consensus

### PR DESCRIPTION
**Pull request by weiihann**

### CODE REVIEW

File: non-consensus-state-expiry/README.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+#### 4. Put the config files in both folders
+```bash
+cp config.toml node/config.toml
+cp genesis.json node/genesis.json
+cp config.toml remoteDB/config.toml
+cp genesis.json remoteDB/genesis.json
+```
+
+#### 5. There are 2 options for syncing
+a. Sync from [snapshot](https://github.com/bnb-chain/bsc-snapshots) (recommended)
```
Comment (0xbundler) [inline non-consensus-state-expiry/README.md:56]: Developers should be reminded that it is best to download the snapshot of pbss, the current state expiry priority supports pbss.

File: non-consensus-state-expiry/README.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+b. Sync from genesis
+```
+./bsc init --datadir node/data-seed --state.scheme path genesis.json
+./bsc init --datadir remoteDB/data-seed --state.scheme path genesis.json
+```
+
+#### 6. Run nodes
+```
+# Run remoteDB
+./bsc --http --config "./remoteDB/config.toml" --rpc.allow-unprotected-txs --allow-insecure-unlock --gcmode full --syncmode full --ws --datadir "remoteDB/data-seed" --metrics --metrics.addr 0.0.0.0 --metrics.port 6061 --pprof --pprof.port 6071 --http.port 8575 --http.corsdomain "*" --maxpeers 500 --history.transactions 0 --metrics.expensive
```
Comment (0xbundler) [inline non-consensus-state-expiry/README.md]: correct remotedb start script is:
```bash
nohup ./geth --http --http.addr 0.0.0.0 --http.vhosts '*' --config "./node/config.toml" --rpc.allow-unprotected-txs --allow-insecure-unlock --gcmode full --ws --datadir "node/server/data-seed/" --metrics --metrics.addr 0.0.0.0 --metrics.port 6061 --pprof --pprof.port 6071 --http.corsdomain "*" --maxpeers 500  --cache 18000  --metrics.expensive --history.transactions 0 --state-expiry --state-expiry.epoch1 31246769 --state-expiry.epoch2 32122769 --state-expiry.period 876000 --state-expiry.remotemode > node/geth-$(date +"%Y%m%d_%H%M").log 2>&1 &
```

need add flags `--state-expiry --state-expiry.epoch1 31246769 --state-expiry.epoch2 32122769 --state-expiry.period 876000 --state-expiry.remotemode`.

Comment (0xbundler) [reply non-consensus-state-expiry/README.md]: may explain epoch params here.

Comment (weiihann) [reply non-consensus-state-expiry/README.md]: the epoch params are explained in the next subsection called "Flags", along with the other state expiry related flags.

File: non-consensus-state-expiry/README.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+cp genesis.json node/genesis.json
+cp config.toml remoteDB/config.toml
+cp genesis.json remoteDB/genesis.json
+```
+
+#### 5. There are 2 options for syncing
+a. Sync from [snapshot](https://github.com/bnb-chain/bsc-snapshots) (recommended)
+```
+# download snapshot
+aria2c -o testnet-geth.tar.lz4 -s14 -x14 -k100M <snapshot-url>
```
Comment (0xbundler) [inline non-consensus-state-expiry/README.md]: Here is a testnet file name.

File: non-consensus-state-expiry/README.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+b. Sync from genesis
+```
+./bsc init --datadir node/data-seed --state.scheme path genesis.json
+./bsc init --datadir remoteDB/data-seed --state.scheme path genesis.json
+```
+
+#### 6. Run nodes
+```
+# Run remoteDB
+./bsc --http --http.addr 0.0.0.0 --config "./remoteDB/config.toml" --rpc.allow-unprotected-txs --allow-insecure-unlock --gcmode full --ws --datadir "remoteDB/data-seed" --metrics --metrics.addr 0.0.0.0 --metrics.port 6061 --pprof --pprof.port 6071 --http.corsdomain "*" --maxpeers 500 --cache 18000  --metrics.expensive --history.transactions 0 --state-expiry --state-expiry.epoch1 31246769 --state-expiry.epoch2 32122769 --state-expiry.period 876000 --state-expiry.remotemode
```
Comment (0xbundler) [inline non-consensus-state-expiry/README.md]: may use nohup to run the program is better.

Comment (0xbundler) [reply non-consensus-state-expiry/README.md]: remove `--cache 18000` flag is better.

**Review by 0xbundler — APPROVED** [review]

**Pull request merged**